### DISCUSSION TITLE
local install of Seqr not scaling

### DISCUSSION THREAD

**Discussion by Abuhaliqa** [broadinstitute/seqr#2819] [Q&A]
Hello,
  I got a higher-end server, and the runtime for Seqr is the same as the lower-end server. I noticed that seqr is not using any of the new server resources. May I know if there are any settings that I can update to improve the runtime for the in-perm seqr install?
Thanks,

**Comment by hanars** [comment]
With no information on what your new "high end" server, theres not much I can do to help you with your configuration. My first recommendation is to try updating the elasticsearch configuration so it has several nodes instead of a single node, as that can be a big performance booster. We also find that if you are using a cloud server, you can get better performance by deploying the different components of seqr to different machines (i.e run seqr in one pod, elasticsearch in another, postgres in another). However, if you just have one big server thats obviously not an option

**Reply by Abuhaliqa** [reply]
sorry for the late replay,
   by high end, I meant 144 cores and a 4TB memory server. I was hoping to find some configuration to allow us to make full usage of the server. And since Docker is not limiting the number of cores that can be used. So perhaps some configuration for Hail or spark could enable further utilization?
   If you need further information, do let me know, and I apologize for lacking tone as it is my first time approaching such an issue.

**Reply by hanars** [reply]
As I suggested above, the first thing you should try is changing your elasticsearch configuration
